test/remote/gateways/remote_ogone_test.rb

```diff
@@ -74,6 +92,55 @@ def test_successful_purchase_with_3d_secure
     assert_equal '46', response.params['STATUS']
     assert_equal OgoneGateway::SUCCESS_MESSAGE, response.message
     assert response.params['HTML_ANSWER']
+    assert_match 'mpi-v1', Base64.decode64(response.params['HTML_ANSWER'])
```

Comment:
👍  Very nice check !!!

Reply:
💬  Comment:

Now that we introduce 3DS v2, maybe is a good idea to indicate in the name of the test, that you are testing v1

Reply:
nice idea